chart.go

```diff
@@ -0,0 +1,132 @@
+package fltk
+
+/*
+#include "chart.h"
+*/
+import "C"
+import "unsafe"
+
+const (
+	BAR_CHART        = iota /* Bar Chart variant */
+	HORBAR_CHART     = iota /* Horizontal Bar Chart variant */
+	LINE_CHART       = iota /* Line Chart variant */
+	FILLED_CHART     = iota /* Fill Line Chart variant */
+	SPIKE_CHART      = iota /* Spike Chart variant */
+	PIE_CHART        = iota /* Pie Chart variant */
+	SPECIALPIE_CHART = iota /* Special Pie Chart variant */
+)
+
+type Chart struct {
+	widget
+}
+
+func NewChart(x, y, w, h int, text ...string) *Chart {
+	c := &Chart{}
+	initWidget(c, unsafe.Pointer(C.go_fltk_new_Chart(C.int(x), C.int(y), C.int(w), C.int(h), cStringOpt(text))))
+	return c
+}
+
+func (c *Chart) Clear() {
+	C.go_fltk_Chart_clear((*C.GChart)(c.ptr))
+}
+
+//Add the data value val with optional label text and color col to the chart.
+//When color not needed just pass zero
+func (c *Chart) Add(val float32, col Color, text ...string) {
+	//TODO: Should i use float32 or better replace it with float64?
```

Comment:
I'd use float64 to use types that are equivalent to the ones used by FLTK. (same below)